src/app/alert/alert.demo.scss

```diff
@@ -1,6 +1,8 @@
-// Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
-// This software is released under MIT license.
-// The full license information can be found in LICENSE in the root directory of this project.
+/*!
```

Comment:
These `/*!` "loud" comments are preserved in the compressed output by the Sass compiler. You **absolutely do not want** to have these on every source file, or the copyright will be written hundreds of times in our final CSS. You want non-loud comments all over the place, and a single loud comment at the top of `main.scss`.